lib/Timber.php

```diff
@@ -528,9 +528,19 @@ public static function compile( $filenames, $data = array(), $expires = false, $
 				);
 			}
 
-			$output = $loader->render($file, $data, $expires, $cache_mode);
+			$output = $loader->render( $file, $data, $expires, $cache_mode );
 		}
 
+		/**
+		 * Replaces the depraceted `timber_compile_result`.
+		 * It adds the posibility to filter the output ready for render.
+		 *
+		 * @since 2.0.0
+		 *
+		 * @param string $output
+		 */
+		$output = apply_filters( 'timber/compile/output', $output );
```

Comment:
Isn’t "output" more related to "echoing" something? But the compile method would return a html instead of echoing it, right? I’d find `timber/compile/result` not that bad. But naming filters is really hard 😅. What does @jarednova as a native English speaker think?

Reply:
I think with

> @todo In case this isn’t deprecated for 2.0.0, update filter hook name

further down in the code I meant that we should use `/` instead of `_` for the hook name separation.